ruby-sinatra/README.md

```diff
@@ -13,6 +13,6 @@ sudo apt update && sudo apt install libpq-dev
 ### Setup
 
 - Install required gems with `bundle install`
-- Initial database with `bundle exec rake db:create`
-- Run migrations `bundle exec rake db:migrate`
+- Initial database with `bundle exec rake db:create` - If the DB doesn't Exists!!!
+- Run migrations `bundle exec rake db:migrate` - Not needed!!
```

Comment:
Why do we have it then?

Reply:
Because refactor is not the scope of this PR

Reply:
Should we write it as a TODO then?

Reply:
No, I just deleted because we are not using it at all